homeassistant/components/lutron_caseta/__init__.py

```diff
@@ -340,6 +359,23 @@ def __init__(self, device, bridge, bridge_device):
         self._bridge_unique_id = serial_to_unique_id(bridge_device["serial"])
         if "serial" not in self._device:
             return
+
+        if "parent_device" in device:
+            # Check if this entity will be a child of an existing device
+            if device_info_by_device_id is not None:
+                parent_device_info = device_info_by_device_id.get(
+                    device["parent_device"], None
+                )
+                if parent_device_info is not None:
```

Comment:
You could use walrus operators here

Reply:
ok, I'll look it up

Reply:
I've simplified it by doing this:

    parent_device_info := device_info_by_device_id.get(